cfme/tests/services/test_provision_stack.py

```diff
@@ -85,16 +84,17 @@ def dialog_name():
 
 @pytest.yield_fixture
 def template(appliance, provider, provisioning, dialog_name, stack):
-    template_type = provisioning['stack_provisioning']['template_type']
-    temp_type = provisioning['stack_provisioning']['template_type_dd']
```

Comment:
not sure - it's all because of UI - we have e.g. "CloudFormation Templates" as accordion item and "Amazon CloudFormation" as selectable option during template creation. I renamed it to `template_group` for accordion, `template_type` for selectable option as it's UI element called that way